doc/guides/CustomStartupGuide.md

```diff
@@ -617,6 +614,7 @@ To stop all EdgeX services (containers), execute the `make down` command. This w
    ```bash
    make clean
    ```
+   >NOTE: As this command deletes all volumes, you will need to rerun the Add Device steps to re-enable your device(s). 
```

Comment:
```suggestion
   >NOTE: As this command deletes all volumes, you will need to rerun the [Add Device](#add-device) steps to re-enable your device(s). 
```

Reply:
This will create a relative link to that section.

Reply:
Added.  Would you know if I can use the <a name=" "> </a> for anchor points instead of pointing to headers?  There are a few instances that I need the user to jump to a reference that is some distance away from the header (due to the images in the section).

Reply:
I'm sorry, I don't know how to implement that.